### PR TITLE
[4.3.0] Upgrade IS and Kernel Versions in carbon-apimgt

### PR DESCRIPTION
**Pull request by msm1992**
- Upgrades kernel version to 4.9.26.alpha and IS components to IS 6.1 compatible versions
- Adds `org.wso2.carbon.identity.organization.management.core.server.feature` and `identity.branding.preference.management feature` to solve p2-profile-gen issues

- Related carbon-apimgt PR : https://github.com/wso2/carbon-apimgt/pull/12243

### CODE REVIEW

**Review by HiranyaKavishani — APPROVED** [review]

**Review by npamudika — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -1306,26 +1316,26 @@
...
-        <carbon.identity-user-ws.version>5.6.1</carbon.identity-user-ws.version>
-        <carbon.identity-inbound-auth-openid.version>5.8.0</carbon.identity-inbound-auth-openid.version>
-        <carbon.identity-local-auth-basicauth.version>6.6.0</carbon.identity-local-auth-basicauth.version>
-        <carbon.identity-outbound-auth-samlsso.version>5.6.2</carbon.identity-outbound-auth-samlsso.version>
-        <carbon.identity-metadata-saml2.version>1.6.1</carbon.identity-metadata-saml2.version>
-        <carbon.identity-data-publisher-application-authentication.version>5.5.1</carbon.identity-data-publisher-application-authentication.version>
-        <carbon.extension.identity.oauth2.grantType.jwt.version>2.1.1</carbon.extension.identity.oauth2.grantType.jwt.version>
-        <carbon.identity.carbon.auth.saml2.version>5.7.2</carbon.identity.carbon.auth.saml2.version>
-        <carbon.identity.auth.version>1.7.1</carbon.identity.auth.version>
+        <carbon.identity.version>5.25.92</carbon.identity.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1319]: 5.25.684

File: pom.xml [hunk 2]
```diff
@@ -1306,26 +1316,26 @@
...
+        <carbon.identity.governance.version>1.8.14</carbon.identity.governance.version>
+        <carbon.identity.event.handler.account.lock.version>1.8.6</carbon.identity.event.handler.account.lock.version>
+        <carbon.identity.event.handler.notification.version>1.7.7</carbon.identity.event.handler.notification.version>
+        <carbon.identity-inbound-auth-oauth.version>6.11.21</carbon.identity-inbound-auth-oauth.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1323]: 6.13.0

File: pom.xml [hunk 3]
```diff
@@ -1306,26 +1316,26 @@
...
+        <carbon.identity-data-publisher-oauth.version>1.6.2</carbon.identity-data-publisher-oauth.version>
+        <carbon.identity-user-ws.version>5.7.2</carbon.identity-user-ws.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1325]: 5.7.5

File: pom.xml [hunk 4]
```diff
@@ -1306,26 +1316,26 @@
...
-        <carbon.identity-data-publisher-application-authentication.version>5.5.1</carbon.identity-data-publisher-application-authentication.version>
-        <carbon.extension.identity.oauth2.grantType.jwt.version>2.1.1</carbon.extension.identity.oauth2.grantType.jwt.version>
-        <carbon.identity.carbon.auth.saml2.version>5.7.2</carbon.identity.carbon.auth.saml2.version>
-        <carbon.identity.auth.version>1.7.1</carbon.identity.auth.version>
+        <carbon.identity.version>5.25.92</carbon.identity.version>
+        <carbon.identity.governance.version>1.8.14</carbon.identity.governance.version>
+        <carbon.identity.event.handler.account.lock.version>1.8.6</carbon.identity.event.handler.account.lock.version>
+        <carbon.identity.event.handler.notification.version>1.7.7</carbon.identity.event.handler.notification.version>
+        <carbon.identity-inbound-auth-oauth.version>6.11.21</carbon.identity-inbound-auth-oauth.version>
+        <carbon.identity-data-publisher-oauth.version>1.6.2</carbon.identity-data-publisher-oauth.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1324]: 1.6.9

File: pom.xml [hunk 5]
```diff
@@ -1459,7 +1469,7 @@
         <config.mapper.version>1.0.2</config.mapper.version>
 
         <!-- Authenticator Properties -->
-        <identity.outbound.auth.oidc.version>5.9.0</identity.outbound.auth.oidc.version>
+        <identity.outbound.auth.oidc.version>5.11.5</identity.outbound.auth.oidc.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1472]: 5.11.32

File: pom.xml [hunk 6]
```diff
@@ -1444,7 +1454,7 @@
         <!-- Forget-me tool -->
         <forgetme.tool.version>1.3.8</forgetme.tool.version>
         <!--Consent Management dependencies-->
-        <carbon.consent.mgt.version>2.4.1</carbon.consent.mgt.version>
+        <carbon.consent.mgt.version>2.5.0</carbon.consent.mgt.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1457]: 5.2.3

File: pom.xml [hunk 7]
```diff
@@ -1306,26 +1316,26 @@
...
+        <carbon.extension.identity.oauth2.grantType.jwt.version>2.2.0</carbon.extension.identity.oauth2.grantType.jwt.version>
+        <carbon.identity.carbon.auth.saml2.version>5.8.3</carbon.identity.carbon.auth.saml2.version>
+        <carbon.identity.auth.version>1.8.11</carbon.identity.auth.version>
         <identity.apps.version>1.5.0</identity.apps.version>
-        <identity.oauth.addons.version>2.4.8</identity.oauth.addons.version>
+        <identity.oauth.addons.version>2.4.11</identity.oauth.addons.version>
 
         <!--SAML Common Utils Version-->
-        <saml.common.util.version>1.2.1</saml.common.util.version>
+        <saml.common.util.version>1.3.0</saml.common.util.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1338]: 1.3.1

File: pom.xml [hunk 8]
```diff
@@ -1306,26 +1316,26 @@
...
+        <carbon.identity-local-auth-basicauth.version>6.7.8</carbon.identity-local-auth-basicauth.version>
+        <carbon.identity-outbound-auth-samlsso.version>5.8.11</carbon.identity-outbound-auth-samlsso.version>
+        <carbon.identity-metadata-saml2.version>1.7.2</carbon.identity-metadata-saml2.version>
+        <carbon.identity-data-publisher-application-authentication.version>5.6.6</carbon.identity-data-publisher-application-authentication.version>
+        <carbon.extension.identity.oauth2.grantType.jwt.version>2.2.0</carbon.extension.identity.oauth2.grantType.jwt.version>
+        <carbon.identity.carbon.auth.saml2.version>5.8.3</carbon.identity.carbon.auth.saml2.version>
+        <carbon.identity.auth.version>1.8.11</carbon.identity.auth.version>
         <identity.apps.version>1.5.0</identity.apps.version>
-        <identity.oauth.addons.version>2.4.8</identity.oauth.addons.version>
+        <identity.oauth.addons.version>2.4.11</identity.oauth.addons.version>
```
Comment (HiranyaKavishani) [inline pom.xml:1335]: 2.4.37

File: pom.xml (hunk 1 above)
Comment (msm1992) [reply pom.xml:1319]: As mentioned in the description this PR is meant to upgrade the IS components to IS 6.1 compatible versions only. Component version upgrade to latest versions is addressed in https://github.com/wso2/product-apim/pull/13353.

File: pom.xml (hunk 2 above)
Comment (msm1992) [reply pom.xml:1323]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 3 above)
Comment (msm1992) [reply pom.xml:1325]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 4 above)
Comment (msm1992) [reply pom.xml:1324]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 5 above)
Comment (msm1992) [reply pom.xml:1472]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 6 above)
Comment (msm1992) [reply pom.xml:1457]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 7 above)
Comment (msm1992) [reply pom.xml:1338]: https://github.com/wso2/product-apim/pull/13353

File: pom.xml (hunk 8 above)
Comment (msm1992) [reply pom.xml:1335]: https://github.com/wso2/product-apim/pull/13353

**Pull request merged**